src/bmark/bmark_solvers.jl

```diff
@@ -69,11 +83,24 @@ if Pkg.installed("BenchmarkProfiles") != nothing
   * A Dict{Symbol, Array{Int,2}} of statistics
   * a profile as returned by `performance_profile()`.
   """
-  function bmark_and_profile(args...;
+  function bmark_and_profile(args...; cost :: Function = stat->stat.elapsed_time,
                              bmark_args :: Dict{Symbol, Any}=Dict{Symbol,Any}(),
                              profile_args :: Dict{Symbol, Any}=Dict{Symbol,Any}())
     stats = bmark_solvers(args...; bmark_args...)
-    profiles = profile_solvers(stats; profile_args...)
+    profiles = profile_solvers(stats, cost=cost; profile_args...)
     return stats, profiles
   end
+
+  bmark_and_profile_by_time(args...;
+                            bmark_args :: Dict{Symbol, Any}=Dict{Symbol,Any}(),
+                            profile_args :: Dict{Symbol, Any}=Dict{Symbol,Any}()) =
+    bmark_and_profile(args..., cost=stat->stat.elapsed_time; bmark_args=bmark_args,
```

Comment:
There may be a problem if `cost` is already in `args` here. But I don't see the point of this function since the default above is to plot a profile by time.